pkg/morph/client/netmap/wrapper/wrapper.go

```diff
@@ -29,26 +29,43 @@ type Wrapper struct {
 // parameter of Wrapper.
 type Option func(*opts)
 
-type opts []client.StaticClientOption
+type opts struct {
+	alphabet bool
```

Comment:
why have you connected all logic with `alphabet`? seems like it is gonna work but it's too implicit IMO

Reply:
No longer relevant